core/src/main/scala/org/broadinstitute/dsde/rawls/model/DataReferenceModel.scala

```diff
@@ -13,7 +17,16 @@ object DataRepoSnapshotReference {
   }
 }
 
+object DataRepoSnapshotListFromDataReferenceList {
+  def apply(ref: DataReferenceList): DataRepoSnapshotList = {
+    val buffer = for (r <- ref.getResources.asScala) yield DataRepoSnapshotReference(r)
```

Comment:
can `getResources` return null here, or would it return an empty collection?

Reply:
WM always returns a DataReferenceList with an empty collection if there's no items